layers/timeline_semaphore.c

```diff
@@ -2183,6 +2186,26 @@ static void timeline_DestroyInstance(
     struct instance_data *instance = object_find(&global_map, _instance);
     PFN_vkDestroyInstance destroy_cb = instance->vtable.DestroyInstance;
 
+    // Unmap physical devices from global_map
+    VkPhysicalDevice* pPhysicalDevices = vk_alloc(&instance->alloc,
+            sizeof(VkPhysicalDevice*) * instance->physicalDeviceCount,
```

Comment:
Should be :
```
sizeof(VkPhysicalDevice) * instance->physicalDeviceCount
```

It's an array of physical device object not an array of pointers of physical devices.

Alternatively you can write : 
```
sizeof(*pPhysicalDevices) * instance->physicalDeviceCount
```

Reply:
Good catch!